.storybook/webpack.config.js

```diff
@@ -42,16 +42,13 @@ module.exports = (baseConfig, env, config) => {
           loader: 'babel-loader',
           options: {
             babelrc: false,
+            configFile: `${__dirname}/.babelrc`,
```

Comment:
I must be missing something… about why this line is here, as it seems to contradict the above line (`babelrc: false`)?

Reply:
Per https://babeljs.io/docs/en/config-files, .babelrc is a file relative config, while the value passed into configFile is project-wide. "configFile" here does the equivalent of copying the contents of .storybook/.babelrc directly into this config object (which is what it looked like prior to this PR) so this is a chance for avoiding repition.

Originally we needed to disable .babelrc files because the root of the repo had one one that wasn't suitable for web compilation, but I was able to remove that as part of #836. So I **think** I can remove that `babelrc: false` line now.